modules/ROOT/pages/supported-cluster-config.adoc

```diff
@@ -69,22 +69,22 @@ You must configure a load balancer to use round-robin distribution of traffic am
 
 You must understand and adhere to the following recommendations and limitations when installing Anypoint Platform PCE and deploying apps.
 
-These limitations are based on a 6-node environment consisting of 6 nodes of 8 cores and 32GB RAM.
+These limitations are based on a 6-node PCE environment consisting of 6 nodes of 8 cores and 32GB RAM.
 
 **** Runtime management
 
-***** Maximum number of servers: 1000
+***** Maximum number of servers across all business groups: 1500 (or 750 Mule runtime clusters, consisting of 2 Mule runtime engines per cluster)
```

Comment:
Do you want to explicitly give an example for 750 2-node clusters here?  If the customer builds clusters with 8 servers, would that be about 187 clusters only?  So the number of clusters is dependent on the cluster size.

Reply:
The intent of that example is to make clear that the limit is on connected runtimes and not other deployment targets like clusters or server groups. This is to avoid confusions upfront of someone connecting 4.5k runtimes and for 1.5k clusters of 3 runtimes each.
If we reword the phrase to add a `for example,...` will make it more clear?

Reply:
Maybe this?

Maximum number of servers across all business groups: 1500 (For example, 750 Mule runtime clusters, consisting of 2 Mule runtime engines per cluster.)

Reply:
Yes.  Adding "for example" instead of "or" will be more clear.

Reply:
Done here https://github.com/mulesoft/docs-private-cloud/pull/168/commits/3e2a090f1d0bad7d6c86053ab272e553bd2e9249